src/clients/python/sams_client/schemas/assets.py

```diff
@@ -39,14 +40,24 @@ class AssetStates(NamedTuple):
         'type': 'string',
         'mapping': not_analyzed
     },
-    '_created_by': {
+    'original_creator': {
```

Comment:
These renames/additions should be changed/added to the schema docstring (so documentation is updated):
https://github.com/superdesk/sams/blob/902a4d2bd59f6b249a630ac9f123286b4863480e/src/clients/python/sams_client/schemas/assets.py#L175